actionmailbox/lib/action_mailbox/errors.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "active_support/actionable_error"
+require "rails/command"
+
+module ActionMailbox
+  # Generic base class for all Action Mailbox exceptions.
+  class Error < StandardError; end
+
+  # Raised when we detect that Action Mailbox has not been initialized.
+  class SetupError < Error
+    include ActiveSupport::ActionableError
+
+    def initialize(message = nil)
+      return super if message
+
+      super("Action Mailbox uses a table in your application’s database named " \
+            "two and active_storage_attachments. To generate it run the following" \
```

Comment:
This exception message has an error in it. We don't have to mention the table names either, that's too low level for the abstraction we're making.

Reply:
Does `action_mailbox:install` run migrations automatically? Otherwise we'd just raise the exception over and over.

Reply:
Yeah, we can improve the UX here. It currently creates a pending migration, which in turn issues another actionable error 😅